src/pages/task/taskTabs/buildBaronAndAnnotations/FileTicketButton.tsx

```diff
@@ -49,23 +49,23 @@ const FileTicketButton: React.VFC<FileTicketProps> = ({
   return (
     <Container>
       <Popconfirm
```

Comment:
strangely this button isn't working for me anymore! I restarted my dev server to make sure 🤔